ceph/rbd/initial_config.py

```diff
@@ -138,7 +138,10 @@ def update_config(**kw):
                                 {"io_total": rep_pool_config.get("io_total", "1G")}
                             )
 
-                        if rep_pool_config.get("mirrormode") != "snapshot":
+                        if (
+                            kw.get("is_mirror")
```

Comment:
what is the purpose of `is_mirror` here? does this new config param?

Reply:
If its a mirroring configuration, then we are enabling journalling feature by default. That's the reason we use this. However for regular RBD tests we don't need to enable this feature. 
So the initial_mirror_config method passes this config so that in case of mirroring we enable this feature.